### ISSUE TITLE
Notebooks create incorrect plots

### ISSUE THREAD

**Issue by victorprincipe** [victorprincipe/skmatter-ore#1]
The Jupyter notebooks in the `paper/` directory are currently _not_ producing the plots that were submitted in the paper (some examples shown below). I used the most recent notebooks stored in [paper-ore](https://github.com/lab-cosmo/scikit-matter/tree/paper-ore) branch of skmatter, but those who worked on each plot likely have the updated notebooks stored locally.

@agoscinski and @rosecers, could you please take a look at the relevant notebooks that you used to create your plots, and check to see whether:
1. You have updated notebooks stored locally
2. Whether the fix is simple and only requires a change to a couple of lines

Please __ignore__ the `Ice.ipynb` at the moment, as I'm still computing some necessary kernels and I need to subsequently alter it slightly so that the notebook can be run without taking a day to compute a 3 GB kernel matrix....

Please open a pull request in this repo for any changes you would like to push.

Thanks! :)

Some example (incorrect) plots that the notebooks create (but there are more):

<img width="376" alt="Screenshot 2023-04-13 at 08 54 06" src="https://user-images.githubusercontent.com/75172693/232225329-e2c83387-ed7e-4575-b54f-1ea37fe21f95.png">
-d6d3cf14cb8c.png">
<img width="663" alt="Screenshot 2023-04-13 at 08 54 20" src="https://user-images.githubusercontent.com/75172693/232225336-81d99550-9790-4ae6-a009-50d67a10acf4.png">
<img width="693" alt="Screenshot 2023-04-13 at 08 54 30" src="https://user-images.githubusercontent.com/75172693/232225344-ee86c96a-dc53-4a8a-b6f3-740b2151c3ae.png">
<img width="635" alt="Screenshot 2023-04-13 at 08 53 58" src="https://user-images.githubusercontent.com/75172693/232225401-b6cf83b6-d20e-4199-b3e8-0c1dd7650e14.png">

**Comment by agoscinski** [comment]
I get the same results. I think its because we do not set the random seed and Linux and Mac have different default seeds (I have the feeling that we already had fixed this once and never pushed it). So using `random_state=0` in `train_test_split`
```python
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.3, shuffle=True, random_state=0)
```
Should fix it for the `WhoDataset-Selection.ipynb`. Can you try it for this notebook and check if the difference still remains?

**Comment by victorprincipe** [comment]
You were absolutely right, with `random_state=0` I get the same plot but with different colours. I've now shuffled the colours around so that the exact figure from the paper is made. I've pushed the updated notebook, but note that the issues with the other notebooks still remain...

**Comment by rosecers** [comment]
What specific issues?

**Comment by agoscinski** [comment]
I think he means the issue that the plots disagree with the ones from the paper. But these notebooks (`WhoDataset-PCovR.ipynb` and `PCovR Charge Model.ipynb`) also contain `train_test_split` without a `random_state` defined, so I believe adding `random_state=0` will fix these issues too.

Btw. I think the file `ice_w_charges_smaller.xyz` is missing for notebook `PCovR Charge Model.ipynb`.

**Comment by agoscinski** [comment]
Should be fixed now with PR #4 and PR #6
